www/klei00/biography/styles/style.css

```diff
@@ -0,0 +1,90 @@
+* {
+    box-sizing: border-box;
+    font-family: Arial;
+}
+body {
+    background-color: black;
+}
+/*---------- Navigation -----------*/
+.navigation {
+    padding: 0;
+    border-bottom: solid 1px white;
+    font-size: 16px;
+    width: 100%;
+}
+nav ul {
+    list-style-type: none;
+    padding: 0;
+    margin: 0 auto;
+    width: 100%;
+    text-align: center;
+}
+nav li {
+    display: inline-block;
+}
+nav a {
+    padding: 10px 20px;
+    text-decoration: none;
+    color: white;
+    display: inline-block;
+}
+nav a:hover {
+    background-color: #253ab3;
+}
+#current_page {
+    padding: 10px 20px;
+    color: white;
+    background-color: #253ab3;
+}
+/* ---------- Main Heading ------------*/
+div.main_heading {
+    padding: 30px;
+    margin: 0;
+    background-image: url("../images/id.png");
+    background-repeat: no-repeat;
+    background-size: 100% 100%;
+    text-align: center;
+    color: white;
+    text-shadow: 0px 0px 1px black, 0px 0px 10px black, 1px 1px 1px black;
+}
+main {
+    background-color: white;
+    margin: 0 auto;
+    max-width: 800px;
+}
+h2, h3, p {
+    margin: 20px 30px;
+}
+h2 {
+    color: #253ab3;
+    font-weight: 800;
+}
+/* ---------- Members ------------ */
+.members {
+    padding: 10px 0 30px 0;
+    border-top: solid 12px black;
+}
+.member {
+    display: inline-block;
+    width: 49.5%;
```

Comment:
možná jsi tu dala 50% ale nefungovalo to - jenom chci vysvětlit v případě, že tě to zajímá:

**display: inline-block** bere v potaz i bílé znaky (nový rádek, mezera, tabulátor) mezi elementy v HTML kódu, pokud by mezi těmi **`<div class="member">`** nebyly vůbec žádné bílé znaky, které zabírají pár pixelů, tak to bude fungovat s 50%. V HTML kódu by byl ale zmatek, proto se používá jedno z:
- float + clear
- flexbox
- grid

V případě **float + clear** stačí dát do `.member` vlastnost `float: left` a přidat tzv. **clear** element na konci seznamu `.members`
```html
<div class="members">
  <div class="member"></div>
  <div class="member"></div>
  <div class="member"></div>
  <div class="clear"></div>
</div>
```

```css
/* float + clear */
.members { }
.member {
  float: left;
  width: 50%;
}
.clear{
  clear: both;
}
```

Stejně tak jako v případě alb, akorát 33.33333% nebo calc(100% / 3) místo 50

My si ale ukážeme lepší způsob pomocí flexboxu (možná i grid)

Reply:
Aha, přesně jsem přemýšlela, proč těch 50 % nejde. Děkuju za vyjasnění :)